docs/docsite/rst/dev_guide/developing_modules.rst

```diff
@@ -68,21 +70,27 @@ Ok, let's get going with an example.  We'll use Python.  For starters, save this
 Testing Modules
 ````````````````
 
-There's a useful test script in the source checkout for Ansible::
+There's a useful test script in the source checkout for Ansible:
+
+.. code-block:: shell-session
 
-    git clone git://github.com/ansible/ansible.git --recursive
-    source ansible/hacking/env-setup
+    $ git clone git://github.com/ansible/ansible.git --recursive
+    $ source ansible/hacking/env-setup
```

Comment:
We decided, no need for the prompt ($) if there's no output (makes it easier to cut and paste commands)